### PR TITLE
VFB-39-Specify visible time period for calendar

### PR DESCRIPTION
**Pull request by mtharkins**
## What's changed
Start and end time has been added to the weekly & daily calendar views 

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| image 
image
  | image 
image
|

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA 
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
  - Will also be automatically checked on pull request
- [x] Make sure you've tested via `npm run test`
  - Will also be automatically checked on pull request

### CODE REVIEW

File: src/components/Calendar/Calendar.tsx [hunk 1]
```diff
@@ -210,6 +210,9 @@ const Calendar: React.FC<CalendarProps> = ({
 }) => {
     const [eventClick, setEventClick] = useState<CalendarEvent | null>(null);
     const calendarRef = useRef<FullCalendar>(null);
+    const [calendarStartTime] = useState("10:00:00");
+    const [calendarEndTime] = useState("15:00:00");
```
Comment (rickaw) [inline src/components/Calendar/Calendar.tsx:214]: If we're not changing these values then no need to have states for them. We can instead do
```
const calendarStartTime = "10:00:00";
```

File: src/components/Calendar/Calendar.tsx [hunk 2]
```diff
@@ -270,6 +273,8 @@ const Calendar: React.FC<CalendarProps> = ({
                                 meridiem: "short",
                             },
                             titleFormat: { year: "numeric", month: "short", day: "numeric" },
+                            calendarStartTime,
+                            calendarEndTime,
```
Comment (rickaw) [inline src/components/Calendar/Calendar.tsx:277]: are there any docs for these props? I'm struggling to find any

File: src/components/Calendar/Calendar.tsx (hunk 1 above)
Comment (mtharkins) [reply src/components/Calendar/Calendar.tsx]: Updated

File: src/components/Calendar/Calendar.tsx (hunk 2 above)
Comment (mtharkins) [reply src/components/Calendar/Calendar.tsx]: slotMinTime/slotMaxTime are now being used instead to set the calendar hours.

**Review by rickaw — APPROVED** [review]
great thanks!

**Pull request merged**